Dockerfile-gamestore

```diff
@@ -0,0 +1,34 @@
+# syntax=docker/dockerfile:1
+
+FROM python:3.8.10-slim-buster
+
+RUN adduser user
+RUN usermod -aG sudo user
+
+WORKDIR /gamestore
+
+COPY requirements.txt requirements.txt
+RUN pip3 install -r requirements.txt
+
+COPY . /gamestore
+
+RUN export GUNICORN_PATH=$(which gunicorn)
+
+RUN apt-get update
+RUN apt-get install -y --reinstall systemd
+
+COPY /gunicorn.socket /etc/systemd/system/gunicorn.socket
+COPY /gunicorn.service /etc/systemd/system/gunicorn.service
+
+
+RUN systemctl start gunicorn.socket
+RUN systemctl enable gunicorn.socket
+
+EXPOSE 8000
+
+CMD ["gunicorn", "--bind", ":8000", "--workers", "3", "gamestore.wsgi:application"]
+
+RUN python3 manage.py createsuperuser -u admin -p 'admin'
+RUN python3 manage.py collectstatic
```

Comment:
You should move `collectstatic` up before `CMD` since you want to collect it before running your application.